core/metamodel/src/main/java/org/apache/isis/core/metamodel/adapter/oid/Oid_Marshaller.java

```diff
@@ -88,14 +87,10 @@
 
     private Oid_Marshaller(){}
 
-    // -- public constants
-    public static final String VIEWMODEL_INDICATOR =
-            Bookmark.ObjectState.VIEW_MODEL.getCode(); // "*"
-
-
-    // -- private constants
-    private static final String TRANSIENT_INDICATOR =
-            Bookmark.ObjectState.TRANSIENT.getCode() ; // "!"
+    @Deprecated
```

Comment:
I wonder why these are required?  The objectType will allow us to determine if this is a view model or an entity, so why this flag also?